bumble/gatt.py

```diff
@@ -325,6 +327,12 @@ def get_descriptor(self, descriptor_type):
 
         return None
 
+    def has_properties(self, properties: List[Any]):
```

Comment:
The type could me more flexible. Accepting an `Iterable` instead of a`List` allows the caller to use tuples or other iterables. Also, shouldn't it be an iterable of `int` rather than `Any` here?